src/kiwixapp.h

```diff
@@ -10,6 +10,7 @@
 #include "kprofile.h"
 #include "settingsmanager.h"
 #include "translation.h"
+#include "downloadmanagement.h"
```

Comment:
Better include in the .cpp file

Reply:
Just for future reference, how should I decide where to include each header file?

Reply:
Try to minimize dependencies (and don't introduce false dependencies). "kiwixapp.h" doesn't depend on "downloadmanagement.h" (the former doesn't use anything declared in the latter). By including "downloadmanagement.h" in "kiwixapp.h" you grow the list of formal dependencies of "downloadmanagement.h". Then if anything changes in "downloadmanagement.h" more code will be recompiled. If you include "downloadmanagement.h" only in the .cpp files that use stuff from that header, then recompilation in case of touching it will be faster.